spec/unit/lib/gateways/fixed_data_gateway_spec.rb

```diff
@@ -0,0 +1,12 @@
+describe FixedDataGateway do
+  let(:fixed_data_gateway) { FixedDataGateway.new }
+
+  let(:fixed_data) { { year: 2011, rc_uplift: 0.00015 } }
+
+  it 'creates new row for year and uplift' do
+    fixed_data_gateway.save(fixed_data)
+    fixed_data = fixed_data_gateway.all
+    expect(fixed_data[0][:rc_uplift]).to eq(0.00015)
+    expect(fixed_data[0][:year]).to eq(2011)
+  end
+end
```

Comment:
I recommend adding an additional example for triangulation, as at the moment the FixedDataGateway could be slimed - containing hardcoded 0.00015 and 2011 values.